graphql/schema/mutation.graphql

```diff
@@ -88,17 +86,19 @@ type Mutation {
   updateVolume(updateVolumeInput: UpdateVolumeInput!): Boolean!
```

Comment:
view/edit hosts is also a permission we should be gating (although its not one we really give special permissions to so low priority); is that ticketed?

Reply:
I think it will be ticketed as the outcome of [DEVPROD-5461](https://jira.mongodb.org/browse/DEVPROD-5461)!